### PR TITLE
Feature/task 561 editorjs send logs to logzio

### PR DESCRIPTION
**Pull request by fabionofre**

### CODE REVIEW

File: dist/editor.licenses.txt [hunk 1]
```diff
@@ -185,6 +185,29 @@ Awesome, nor vice versa. **Please do not use brand logos for any purpose except
 to represent the company, product, or service to which they refer.**
 
 
+axios
```
Comment (gabrielmoterani) [inline dist/editor.licenses.txt:188]: ??

Comment (gabrielmoterani) [reply dist/editor.licenses.txt:188]: Oh, axios

**Review by gabrielmoterani — APPROVED** [review]

**Comment by gabrielmoterani** [comment]
Hey @pepeladeira and @fabionofre. We need to sync with the new repo

**Review by pepeladeira — APPROVED** [review]
Good!

**Comment by pepeladeira** [comment]
> Hey @pepeladeira and @fabionofre. We need to sync with the new repo

We will create the shakespeare.js repository when we merge our `editor-minor-improvements` branch, because doing in that way, we will not lost this branch

**Review by matheusslg — APPROVED** [review]

**Pull request merged**